VirtoCommerce.Platform.Core/Jobs/JobPriority.cs

```diff
@@ -0,0 +1,9 @@
+namespace VirtoCommerce.Platform.Core.Web.Jobs
```

Comment:
I wonder to know, how to then configure Hangfire priority of resources for this named queues? Any link to documentation will be appreciated!

Reply:
Hangfire will process jobs in order of the queues: http://docs.hangfire.io/en/latest/background-processing/configuring-queues.html

Reply:
Thanks. I've looked at this article, but I missed this line  'The order is important, workers will fetch jobs from the critical queue first, and then from the default queue.'